src/components/shared/Tooltip.js

```diff
@@ -10,7 +10,7 @@ import type { CssPixels } from '../../types/units';
 import { ensureExists } from '../../utils/flow';
 require('./Tooltip.css');
 
-const MOUSE_OFFSET = 21;
+const MOUSE_OFFSET = 11;
```

Comment:
because with this patch we now apply the offset both vertically and horizontally, I reduce it so that the tooltip stays close enough to the cursor.